test/py/libvfio_user.py

```diff
@@ -89,10 +89,10 @@
 VFIO_DEVICE_FLAGS_RESET = (1 << 0)
 VFIO_DEVICE_FLAGS_PCI = (1 << 1)
 
-VFIO_REGION_INFO_FLAG_READ = (1 << 0)
```

Comment:
do you still need/want these changes?

Reply:
We don't need them, I forgot to remove them.